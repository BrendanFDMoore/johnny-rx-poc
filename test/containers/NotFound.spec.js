import React from 'react';
import { mount } from 'enzyme';

import NotFound from 'containers/NotFound';

function setup() {
  return mount(<NotFound />);
}

describe('NotFound', () => {
  const wrapper = setup(true);

  it('should be a StatelessComponent', () => {
    expect(wrapper.instance().constructor.name).toBe('StatelessComponent');
  });

  it('should render properly', () => {
    expect(wrapper.html()).toMatchSnapshot();
  });
});
