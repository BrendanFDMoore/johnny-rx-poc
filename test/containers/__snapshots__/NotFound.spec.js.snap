exports[`NotFound should render properly 1`] = `"<div class=\"app__not-found app__route\"><h1>404</h1></div>"`;
